src/app/forms/draft-court-order-conviction/draft-court-order-conviction.web.component.js

```diff
@@ -0,0 +1,70 @@
+import React from "react";
+import FormThatPrints from "../inputs/form-that-prints.component.js";
+import TextInput from "../inputs/text-input.component.js";
+import TextArea from "../inputs/text-area.component.js";
+import Select from "../inputs/select.component.js";
+import Checkbox from "../inputs/checkbox.component.js";
+import Radio from "../inputs/radio.component.js";
+import { formCommonOptions } from "../form-common-options/form-common-options";
+
+export default class DraftCourtOrderConviction_Web extends React.Component {
+  render() {
+    return (
+      <FormThatPrints>
+        <TextInput dataKey="person.lastName" label={__("last name")} />
+
+        <TextInput dataKey="person.firstName" label={__("first name")} />
+
+        <TextInput dataKey="person.middleName" label={__("middle name")} />
+
+        <TextInput dataKey="person.addressStreet" label={__("street")} />
+
+        <TextInput dataKey="person.addressCity" label={__("city")} />
+
+        <TextInput dataKey="person.addressState" label={__("state")} />
+
+        <TextInput dataKey="person.addressZip" label={__("zip")} />
+
+        <TextInput dataKey="person.dayPhone" label={__("phone")} />
+
+        <TextInput dataKey="person.email" label={__("email address")} />
+
+        <Radio
+          dataKey="case.courtType"
+          label="Court Type"
+          options={formCommonOptions.courtTypeOptions}
```

Comment:
When you say `formCommonOptions` here, they are common to a single case, right? maybe we should save this as `case.xxx`? What do you think?

Reply:
Form common options would be the options that go into the select, radio, or checkbox components as props. The counties, court type etc...

Reply:
👍 got it